### ISSUE TITLE
Refresh Interval

### ISSUE THREAD

**Issue by russbakr** [mdesilva/AmazonFlexUnlimited#66]
Default :

 "refreshInterval": 0.2,
 
 like that would be able to run the program for what - 10 minutes before rate/limit is reached ?

anyone experiment with differing refreshInterval #'s to see how long they can keep the program running before rate/limit is reached ?

**Comment by ransel2016** [comment]
Yes, I also realized that at that rate it only lasts running for 10 min and the second time it takes a 60 min pause interval and after 90 min, you have to adjust little by little

**Comment by gmcsnetwork** [comment]
@russbakr 

I have been testing for a few days with different "refreshInterval" values, and anything under 1sec gets "Rate limit reached" fairly quickly, anywhere between 10-30mins. Setting the "refreshInterval" to 1.2secs allows the script to run longer 30-60mins before it reaches the first limit and waits for 30 mins.

What have you experienced so far?

**Comment by MY20-PHEV** [comment]
Other scripts I've seen use 5 second refresh rates and stay running 24/7

**Comment by Sierpex** [comment]
I have tested it with 1.2 second interval and it only runs less than 3 minutes

INFO: Starting job search...
0 requests attempted

50 requests attempted

INFO: Job search cycle ending...
INFO: Accepted 0 offers in 135.03191685676575 seconds

**Comment by sack2116** [comment]
> 1.2

is this refresh interval  setting in the cofig file measured in ticks? or standard seconds..
if in ticks and you want 1.2 seconds does that mean i need to convert that to ticks which is 6024 ticks? and use that in the config file?

**Comment by sack2116** [comment]
also how does that relate to the retry limit setting, and what is the optimal value for retrylimit?
